package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vsts-task-tool-lib",
-  "version": "0.9.0",
+  "version": "0.9.1",
```

Comment:
Can bump minor, 0.10.0 since it's a new feature.

Reply:
Done.